aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -1217,13 +1221,13 @@ Tensor sspaddmm(const Tensor& self, const Tensor& mat1, const Tensor& mat2,
 // for ops like sum, max, and min.
 // --------------------------------------------------------------------
 Tensor _sparse_sum(const SparseTensor& input) {
-  return input.coalesce().values().sum();
+  return input.values().sum();
 }
 
 Tensor _sparse_sum(const SparseTensor& input, ScalarType dtype) {
   // don't have to do a conversion to the correct dtype first
   // just need to setup the accumulator correctly
-  return input.coalesce().values().sum(dtype);
+  return input.values().sum(dtype);
```

Comment:
I've  checked  the tests and they fail with these changes, and it's not required to remove the memory leaks. I think  `.coalesce()` should be maintained, if not, there will not be possible to sum values over a non coalesce tensor.